packages/styles/docs/sass.md

```diff
@@ -89,10 +99,23 @@ between version updates.
 
 ## Motion
 
+The motion package provides helper functions, mixins, and duration tokens to add motion into your project.
+
+You can bring in the motion package and tokens by using `@use`. 
+
+For example:
+
+```scss
+ @use '@carbon/styles/scss/motion';
+ .dropdown--arrow {
+  transition: transform $duration-fast-02 motion(standard, productive);
```

Comment:
I think the snippet will need to be updated to:

```scss
@use '@carbon/styles/scss/motion' as *;
```

For those variables to be defined.